app/main.py

```diff
@@ -1,6 +1,20 @@
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+FORMAT = "json"
+BASE_URL = "http://api.weatherapi.com/v1"
+FILTERING = "Paris"
```

Comment:
Consider renaming the `FILTERING` constant to something more descriptive, like `CITY_NAME`, to better convey its purpose. This aligns with the task requirement to get the current weather for Paris city.